handout/testing/unit_testing_redux.md

```diff
@@ -0,0 +1,229 @@
+# Unit Testing Redux
+
+The units that make up redux are actions and reducers. Lets try testing them.
```

Comment:
Maybe something more expansive, like:

Unit testing Redux is a very straightforward process.  There are two units, reducers, and actions.  Reducers are pure functions that lend themselves well to testing.  Actions trigger changes in a Redux system, there are two broad categories of actions, synchronous, and asynchronous.  Synchronous actions are quite straightforward to test, but asynchronous actions are slightly more involved.  The examples below should provide the reader with a strong foundation for testing Redux applications.